examples/authentication/prisma.yml

```diff
@@ -0,0 +1,8 @@
+endpoint: http://localhost:4466/authentication
+datamodel: datamodel.graphql
+
+hooks:
+  post-deploy:
+    - echo "Deployment finished"
+    - graphql get-schema --project db
+    - graphql prepare
```

Comment:
prepare can't be used with prisma-binding 2.0, change to codegen

Reply:
noted, updating this, but codegen is also not working at the moment, working on that as well.